scripts/index.js

```diff
@@ -145,3 +145,5 @@ function test() {
 }
 
 test();
+
+
```

Comment:
Extra white space but well, we might end up changing the entire file anyway so it doesn't matter.